eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -57,14 +57,14 @@ def start(queue, options = {}, &block)
                                                           queue_url: q,
                                                           max_number_of_messages: 1,
                                                           wait_time_seconds: 1,
-                                                          message_attribute_names: ['ApproximateReceiveCount']
+                                                          attribute_names: ['ApproximateReceiveCount']
```

Comment:
According to the AWS SQS documentation here: http://docs.aws.amazon.com/sdkforruby/api/Aws/SQS/Client.html this is not the correct way to request attribute names:

`resp = client.receive_message({
  queue_url: "String", # required
  attribute_names: ["Policy"], # accepts Policy, VisibilityTimeout, MaximumMessageSize, MessageRetentionPeriod, ApproximateNumberOfMessages, ApproximateNumberOfMessagesNotVisible, CreatedTimestamp, LastModifiedTimestamp, QueueArn, ApproximateNumberOfMessagesDelayed, DelaySeconds, ReceiveMessageWaitTimeSeconds, RedrivePolicy
  message_attribute_names: ["MessageAttributeName"],
  max_number_of_messages: 1,
  visibility_timeout: 1,
  wait_time_seconds: 1,
})`

The previous way is the correct way, curious as to what drove this change?